pkg/controller/service/service.go

```diff
@@ -18,18 +18,21 @@ func RenderServices(req router.Request, resp router.Response) error {
 		return err
 	}
 	resp.Objects(objs...)
+	svcInstance.Status.HasService = len(objs) > 0
 
 	objs, err = publish.ServiceLoadBalancer(req, svcInstance)
 	if err != nil {
 		return err
 	}
 	resp.Objects(objs...)
+	svcInstance.Status.HasService = svcInstance.Status.HasService || len(objs) > 0
```

Comment:
Would setting `svcInstance.Status.HasService = len(objs) > 0` at the end of the handler enough?

Reply:
hmm, at a glance, I _think_ you'd get a false negative if the first `objs` returns `len(objs) > 0` but the second does not.

Reply:
No because the second `objs, err =` overwrites the first. So, if the `objs` produces a service, then the next call could overwrite it to nothing.

Reply:
Oh, make sense! For some reason I thought objs are additive and was alwasy appended